### PR TITLE
Prepare v0.4 release

### PR DESCRIPTION
**Pull request by mkelley**
* Updated change log
* Verified and updated URLs as needed
* Updated sub-module status page.
* Set a release date.

### CODE REVIEW

**Comment by mkelley** [comment]
@jianyangli This is ready for review.  I set the release date to today, but that's not important.  You can review whenever you have an opportunity.

File: CHANGES.rst [hunk 1]
```diff
@@ -21,34 +23,61 @@ sbpy.activity
...
+- Index page has been reorganized.
+
+
+API Changes
+-----------
+
+sbpy.data
+^^^^^^^^^
+
+- IAU HG series functions moved from `sbpy.data.core` to `sbpy.data.iau`.
```
Comment (jianyangli) [inline CHANGES.rst:64]: This change is in photometry:

... moved from `sbpy.photometry.core` to `sbpy.photometry.iau`

File: CHANGES.rst [hunk 2]
```diff
@@ -21,34 +23,61 @@ sbpy.activity
...
 - Added parameter constraints to the IAU disk-integrated phase function models,
   such as ``HG``, ``HG1G2``, ``HG12``, and ``HG12_Pen16``. [#366]
 
+Documentation
+^^^^^^^^^^^^^
+
+- Index page has been reorganized.
+
+
+API Changes
```
Comment (jianyangli) [inline CHANGES.rst:58]: There is a duplicated API change entry in line 92.  Should they be merged?

File: docs/install.rst [hunk 3]
```diff
@@ -12,7 +12,7 @@ care of with installation using pip:
 * `numpy <https://numpy.org/>`__ 1.17.0 or later
```
Comment (jianyangli) [inline docs/install.rst:12]: Version numbers of various packages (including python) need to be updated here as well.

**Review by jianyangli — COMMENTED** [review]
@mkelley A few places in CHANGES.rst and install.rst need to be updated.  Otherwise all look good to me.

File: CHANGES.rst (hunk 1 above)
Comment (mkelley) [reply CHANGES.rst:64]: Fixed.  Also added the PR number.

File: docs/install.rst (hunk 3 above)
Comment (mkelley) [reply docs/install.rst:12]: Updated.

**Review by jianyangli — APPROVED** [review]

**Pull request merged**